apps/iotune/iotune.cc

```diff
@@ -549,6 +565,12 @@ class iotune_multi_shard_context {
         });
     }
 
+    future<> force_buffer_size(unsigned buffer_size) {
```

Comment:
I think it makes sense to route the buffer size via iotune_multi_shard_context constructor, next to evaluation_directory argument

Reply:
Done